Source/model/draw/ColorMatrixEffect.swift

```diff
@@ -0,0 +1,59 @@
+import Foundation
+
+open class ColorMatrixEffect: Effect {
+
+    public let matrix: [Double]
+
+    public init(matrix: [Double] = [], input: Effect? = nil) {
```

Comment:
As I understand empty matrix will break in the renderer, right? Do not use such a default value. It should be reasonable or no default value at all.